internal/ethereum/deploy_contract_prepare_test.go

```diff
@@ -54,6 +55,37 @@ const samplePrepareDeployTX = `{
 	"params": [ 4276993775 ]
 }`
 
+const samplePrepareDeployTXLargeInputParams = `{
+	"ffcapi": {
+		"version": "v1.0.0",
+		"id": "904F177C-C790-4B01-BDF4-F2B4E52E607E",
+		"type": "DeployContract"
+	},
+	"from": "0xb480F96c0a3d6E9e9a263e4665a39bFa6c4d01E8",
+	"to": "0xe1a078b9e2b145d0a7387f09277c6ae1d9470771",
+	"gas": 1000000,
+	"nonce": "111",
+	"value": "12345678901234567890123456789",
+	"contract": "0xdeadbeef",
+	"definition": [{
+		"inputs": [
+			{
+				"internalType":" uint256",
+				"name": "y",
+				"type": "uint256"
+			},
+		    {
+			    "internalType":" string",
+			    "name": "x",
+			    "type": "string"
+		    }
+		],
+		"outputs":[],
+		"type":"constructor"
+	}],
+	"params": [ 10000000000000000000000001, "some-text" ]
```

Comment:
This input is broken in JSON, it's attempting to use a number that does not fit into int64